validation/dam_break_2d/validation_dam_break_2d.jl

```diff
@@ -95,22 +95,26 @@ viscosity_edac = ViscosityAdami(nu=alpha * smoothing_length * sound_speed / 8)
 fluid_system_edac = EntropicallyDampedSPHSystem(tank_edac.fluid, smoothing_kernel,
                                                 smoothing_length,
                                                 sound_speed, viscosity=viscosity_edac,
+                                                density_calculator=SummationDensity(),
                                                 acceleration=(0.0, -gravity))
 
 trixi_include(@__MODULE__, joinpath(examples_dir(), "fluid", "dam_break_2d.jl"),
               fluid_particle_spacing=particle_spacing,
-              smoothing_length=3.5 * particle_spacing,
+              smoothing_length=smoothing_length, smoothing_kernel=smoothing_kernel,
```

Comment:
Can you please change the name to something other than `smoothing_length` and `smoothing_kernel`? Like `smoothing_length_` and `smoothing_kernel_` for example. Otherwise, we might run into the same problem as in the dam break corrections example, where we later add a `trixi_include` without setting smoothing length and kernel, which will then overwrite the variables defined here.